### PR TITLE
Jshift kit thorntail

### LINKED ISSUE #5: Extract Thorntail generator and enricher in a Github repo
Extract Thorntail generator and enricher in a Github repo : `jshift-kit-thorntail`

### PR DESCRIPTION
**Pull request by devang-gaur**
fixes #5

### CODE REVIEW

File: jshift-kit-spring-boot/src/main/java/io/jshift/springboot/watcher/SpringBootWatcher.java [hunk 1]
```diff
@@ -42,6 +28,19 @@
...
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicBoolean;
+
 import static io.jshift.kit.common.util.SpringBootConfigurationHelper.DEV_TOOLS_REMOTE_SECRET;
 
 public class SpringBootWatcher extends BaseWatcher {
```
Comment (lordofthejars) [inline jshift-kit-spring-boot/src/main/java/io/jshift/springboot/watcher/SpringBootWatcher.java]: PR says that we are migrating Thorntail but it seems that also Spring Boot. Now it is ok but in the future let's try to stick on  what description of PR says.

**Review by lordofthejars — APPROVED** [review]

File: jshift-kit-spring-boot/src/main/java/io/jshift/springboot/watcher/SpringBootWatcher.java (hunk 1 above)
Comment (rohanKanojia) [reply jshift-kit-spring-boot/src/main/java/io/jshift/springboot/watcher/SpringBootWatcher.java]: It doesn't seem to be migrated, just some imports coming from docker-maven-plugin

File: jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java [hunk 2]
```diff
@@ -32,22 +48,6 @@
 import java.util.zip.ZipInputStream;
 import java.util.zip.ZipOutputStream;
```
Comment (rohanKanojia) [inline jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java]: What does this change do?

**Review by rohanKanojia — APPROVED** [review]

File: jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java (hunk 2 above)
Comment (devang-gaur) [reply jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java]: Apparently nothing much. It was intellij

Comment (devang-gaur) [reply jshift-kit-spring-boot/src/main/java/io/jshift/springboot/generator/SpringBootGenerator.java]: Fixed

**Pull request merged**